### ISSUE TITLE
[Redesign] Improve filter for accessibility audit

### ISSUE THREAD

**Issue by hidde** [w3c/wcag-em-report-tool#374]
(To be updated)

Suggestions:

- filters separated from heading with whitespace 
- WAI colors ‘cloudy subtle’ and ‘WAI green’ used
- sufficient space around filter choices so that it works for people on touch screens and/or with certain disabilities
- stands out from other parts of the tools
- simplified WCAG version (takes out “Added in WCAG x” choice for futureproofing, as with WCAG 2.2 we’d end up having 2 of those)   _There is no consensus for this, needs further discussion_

<img width="640" alt="screenshot of the above, desktop" src="https://user-images.githubusercontent.com/178782/107507722-474b7900-6ba0-11eb-8cee-6c0de06f20a6.png">
<img width="332" alt="screenshot of the above, mobile" src="https://user-images.githubusercontent.com/178782/107507726-487ca600-6ba0-11eb-97f8-e1da6140717c.png">

**Comment by nitedog** [comment]
I like this direction very much!

As discussed, we need options to:

1. Show all Success Criteria in WCAG 2.0
2. Show all Success Criteria in WCAG 2.1
3. Show all Success Criteria in WCAG 2.2
4. Show Success Criteria in WCAG 2.1 and not in WCAG 2.0
5. Show Success Criteria in WCAG 2.2 and not in WCAG 2.0 (or changed)
6. Show Success Criteria in WCAG 2.2 and not in WCAG 2.1 (or changed)

**Comment by hidde** [comment]
That's a lot of options. 

If we really want that many options, I'd suggest we use a `select` element with “Show all Success Criteria in [latest published  version]” as the default so that most people don't need it.

One other way I could imagine is, having the first three displayed, and an “Advanced” button, which opens an overlay to select the more complex options (4 to 6 in your list).

**Comment by rvantonisse-accessibility** [comment]
Hey guys,

Also one potential filter gotcha:

When selecting wcag 2.0 in scope step as target version, do we then still want to show the version filter? It would not make sense to have a wcag audit done on 2.2 criteria when 2.0 is targeted for whatever reason.

For this reason it might also be a good idea to mention the wcag version that is being audited / evaluated.

This is different from conformance levels.

Concerning the amount of options; I agree with Hidde to minimize these. I would just pick the filters that point out any differences / new additions from 2.0 until the targeted version like "added in 2.1". This way it is just one option for each new version.

**Comment by nitedog** [comment]
Yes, this is complex and we might need to try several different approaches until we find the right one. I like the "advanced" idea. Generally we might not have one widget (ie. select or options etc.) but rather a combination. And yes, we need to consider the conformance level as well, at least as a default presentation...

**Comment by hidde** [comment]
Latest UI design update: rather than giving the filters their own 'box'  and making it more prominent with a blue background, I tried making it _less_ prominent, by giving it the only the most basic visual treatment. At the same time, I moved it to the left of the audit view, which I feel establishes a clearer relationship. This is a common pattern for filtering in e-commerce: the _filters_ on the left, the _filtered_ on the right.

<img width="963" alt="screenshot of what's described above" src="https://user-images.githubusercontent.com/178782/114531976-aa16ca80-9c4c-11eb-97af-9f7d3d7cc42c.png">
